modules/gapi/src/backends/cpu/gcpustereo.cpp

```diff
@@ -0,0 +1,43 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2021 Intel Corporation
+
+#include "precomp.hpp"
+#include "gnnparsers.hpp"
+
+#include <opencv2/gapi/stereo.hpp>
+#include <opencv2/gapi/cpu/stereo.hpp>
+#include <opencv2/gapi/cpu/gcpukernel.hpp>
+
+#include <opencv2/calib3d.hpp>
+
+GAPI_OCV_KERNEL(GCPUStereoBM, cv::gapi::stereo::GStereoBM)
+{
+    static void run(const cv::Mat& left,
+                    const cv::Mat& right,
+                    const bool bypass,
+                    const int dtype,
+                    const int numDisparities,
+                    int blockSize,
+                    const double baseline,
+                    const double focus,
+                    cv::Mat& out_disp)
+    {
+        cv::Mat disp;
+        cv::StereoBM::create(numDisparities, blockSize) -> compute(left, right, disp);
```

Comment:
Spaces before and after access operator bring confusion in reading

Reply:
Done